test/unit/openmrs.spec.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+/* jasmine specs for controllers go here */
+describe('Concept dictionary controllers', function() {
+
+    beforeEach(function(){
+        jasmine.addMatchers({
+            toEqualData: function(util, customEqualityTesters) {
+                return {
+                    compare: function(actual, expected) {
+                        var passed = angular.equals(actual, expected);
+                        return {
+                            pass: passed,
+                            message: 'Expected ' + actual + (passed ? '' : ' not') + ' to equal ' + expected
+                        }
+                    }
+                }
+            }
+        });
+    });
+
+    beforeEach(module('conceptDictionaryApp'));
+
+    describe('openmrsRest', function(){
```

Comment:
describe should indicate what you are testing, e.g. "listFull should query and return full representations"